pom.xml

```diff
@@ -322,4 +323,17 @@
     </pluginManagement>
   </build>
 
+  <reporting>
```

Comment:
Why did you need to add this?

Reply:
It was removed in `hotels-oss-parent`

Reply:
Ah yes, it doesn't work for Java 8 but since this is Java 7 (for now) we still need it here, OK.